.github/DEV_GUIDE.md

```diff
@@ -105,7 +122,32 @@ Here's the changes I'm making for the purpose of this guide:
 
 Now, save your changes. If we look at the Source Control tab, we'll see that we have some new changes. Git has found every change you made to your fork's repo on your computer! Even if you change a single space in a single line of code, Git will find that change. Just make sure you save your files.
 
-### Step 3: Up On Stage :movie_camera: 
+### Step 3: Testing your code :game_die:
+
+The easiest way to test your changes is to press F5. This compiles your code, runs the server and connects you to it, as well as automatically giving you admin permissions. It also starts a debugger that will let you examine what went wrong when a runtime error happens.
```

Comment:
Mention CTRL+F5 (no debugger DS launch)?

Reply:
I had no idea about that one, thanks!